lib/main.js

```diff
@@ -5,14 +5,31 @@ const parseParams = require('./utils').parseParams
 
 function Busboy (opts) {
```

Comment:
maybe we should convert to proper class as well, and get rid of inherits?

Reply:
can we ensure, that the functionality is still the same?

Reply:
tbh I would prefer to keep it as is for now and do the whole transform ationto real Streams Classes with all pseudoclasses for the next iteration.

Reply:
Also could be breaking, because now you can create Busboy instances without new Operator. And changing to a class means you are forced to use new Operator, right?

Reply:
yeah, good point